Assets/Plugins/Source/Editor/Utility/ScriptingDefineUtility.cs

```diff
@@ -23,64 +23,57 @@
 using UnityEditor;
 using UnityEditor.Build.Reporting;
 
-namespace PlayEveryWare.EpicOnlineServices.Editor.Build
+namespace PlayEveryWare.EpicOnlineServices.Editor.Utility
 {
-    public class EOSPreprocessUtilities
+    using System.Linq;
+
+    public class ScriptingDefineUtility
     {
-        private static bool isScriptingDefineEnabled(string scriptingDefineSymbols, string defineAsString)
+        private static bool DoesScriptingDefineExist(string scriptingDefineSymbols, string defineAsString)
         {
-            foreach (string define in scriptingDefineSymbols.Split(';'))
-            {
-                if (define == defineAsString)
-                {
-                    return true;
-                }
-            }
-
-            return false;
+            return scriptingDefineSymbols.Split(';').Any(define => define == defineAsString);
         }
 
         //-------------------------------------------------------------------------
-        public static bool isScriptingDefineEnabled(BuildReport report, string defineAsString)
+        private static bool DoesScriptingDefineExist(BuildReport report, string defineAsString)
         {
 #if UNITY_2021_2_OR_NEWER
             var namedBuildTarget =
                 UnityEditor.Build.NamedBuildTarget.FromBuildTargetGroup(report.summary.platformGroup);
             var defineSymbolsForGroup = PlayerSettings.GetScriptingDefineSymbols(namedBuildTarget);
 #else
-        // Ensure that we don't do checks for config files if the current platform group has EOS disabled
-        var defineSymbolsForGroup = PlayerSettings.GetScriptingDefineSymbolsForGroup(report.summary.platformGroup);
+            // Ensure that we don't do checks for config files if the current platform group has EOS disabled
```

Comment:
Are those tabs in the file?

Reply:
Something like that. I think I need to fix my `git.autocrlf` setting or something. When I turn off whitespace difference rendering there are no changes according to GitHub.